launcher/modManager/cmodlist.cpp

```diff
@@ -297,12 +340,12 @@ QVector<QString> CModList::getModList() const
 	{
 		for(auto it = repo.begin(); it != repo.end(); it++)
 		{
-			knownMods.insert(it.key());
+			knownMods.insert(it.key().toLower());
```

Comment:
Looks like this construction is repeated a few times and its goal is to get lowered keys
Probably something like vstd::concatenate(knownMods, getLoweredKeys(repo)) might be possible

Reply:
And above keyNormalize = getLoweredKeys(input.toMap())

Reply:
those are QStrings and it's needed to use QT methods to work with them